atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+from flask import Response, current_app, request
+
+import atlassian_jwt_auth
+from atlassian_jwt_auth.contrib.flask_app.utils import parse_jwt
+
+
+def requires_asap(f):
+    """
+    Wrapper for Flask endpoints to make them require authentication to access.
```

Comment:
I suggest changing `Wrapper for Flask endpoints to make them require authentication to access.` to `Wrapper for Flask endpoints to make them require asap authentication to access.`